app/web_hub.go

```diff
@@ -64,10 +64,11 @@ func TotalWebsocketConnections() int {
 }
 
 func HubStart() {
-	l4g.Info(utils.T("api.web_hub.start.starting.debug"), runtime.NumCPU()*2)
+	numberOfHubs := runtime.NumCPU() * 2
+	l4g.Info(utils.T("api.web_hub.start.starting.debug"), numberOfHubs)
 
 	// Total number of hubs is twice the number of CPUs.
```

Comment:
hmm...where?

The line is almost identical to before.  I just pull out `runtime.NumCPU()*2`.